docs/sources/flow/reference/cli/run.md

```diff
@@ -31,6 +31,9 @@ The following flags are supported:
 * `--server.http.ui-path-prefix`: Base path where the UI will be exposed (default `/`).
 * `--storage.path`: Base directory where components can store data (default `data-agent/`).
 * `--disable-reporting`: Disable [usage reporting][] of enabled [components][] to Grafana (default `false`).
+* `--cluster.enabled`: Start the Agent in clustered mode (default `false`).
+* `--cluster.join-address string`: Address to join the cluster at (default `""`).
+* `--cluster.advertise-address`: Address to advertise to other cluster nodes (default `""`).
```

Comment:
Not sure why we have default values for `--cluster.advertise-address` and `--cluster.join-address string`, when empty strings wouldn't work.

Reply:
If `advertise-address` is empty, then an address will be exported from `DefaultInterfaces = []string{"eth0", "en0"}`. If `join-address` is empty, then it supposes there are no peers to connect to and bootstraps a new cluster.

I suppose this was not clear from the doc, so I'm amending them to fix this.